### ISSUE TITLE
Can we add more write-access maintainers to the repo to keep updates flowing?

### ISSUE THREAD

**Issue by caffeinatedMike** [pallets-eco/flask-admin#2153]
Currently there are 50 open pull requests and 340 open issues. I notice that the last commit was from August 2nd and the last official release to PyPI was April 17th. To many people these factors may make the project appear dead and unmaintained. @mrjoes  would you consider granting a select group (maybe a few volunteers from the top contributors?) of additional users write access to the project, so the backlog can be worked on and updates can be more-regularly pushed?

**Comment by mrjoes** [comment]
For sure. Currently, all of our maintainers are fairly inactive - we have more than one.

**Comment by caffeinatedMike** [comment]
I can give it a shot if you'd like. I'm utilizing the bootstrap4 template for a current project, so I'd say my level of knowledge with the extensive code-base is limited. However, if it means keeping the project alive I'll try to get some of the existing PRs merged.

**Comment by michaelbukachi** [comment]
@caffeinatedMike I don't mind helping out. We are using Flask-Admin heavily at work so it would be a huge benefit for us.

**Comment by caffeinatedMike** [comment]
@mrjoes 😄

**Comment by michaelbukachi** [comment]
> Who's there? smile
> 
> Jokes aside, I'm totally OK merging things almost daily as long as someone can veto and sign off on pull requests. I don't have time to do code reviews these days, and it's hard for me to reason if changes are good enough to be accepted. If this'll work out as a start - we can definitely figure out merge permissions.

I don't mind doing code reviews :raising_hand_man:

**Comment by mrjoes** [comment]
Great then. Please stamp PRs with seal of your approval and tag me in comments - I'll start merging things.

Also, I can do release whenever - I know some guys are waiting for some fixes to be released. Let's try this new process and if it'll work - we can do another release soon-ish.

**Comment by caffeinatedMike** [comment]
I'm also open to doing code-reviews and veto/sign-offs. I pretty much already started with my work above 😄

**Comment by caffeinatedMike** [comment]
I dove right in, sorry in advance for all the mentions guys!

**Comment by caffeinatedMike** [comment]
@TitaniumHocker I'm just happy to help/be able to contribute. Unfortunately, I don't have any experience with `flask-babel(x)`. So, I'll need to cede those updates to someone else. Care to pitch in? :smiley:

**Comment by caffeinatedMike** [comment]
@michaelbukachi I definitely second GH Actions. By all means have at it!

**Comment by michaelbukachi** [comment]
> @michaelbukachi I definitely second GH Actions. By all means have at it! Tomorrow I'll look into the two issues I mentioned, as well as continue the ongoing correspondence with some of the open issues. I've addressed a few today and am waiting on a few OPs to reply or implement the requested changes for PRs. Slowly but surely we'll chip away smile

Noice. Looking forward to the day we get those PRs and issues to double digits.

**Comment by ulgens** [comment]
Hey guys, what is the best way to contribute to the cleaning? I can contribute to cleaning the test matrix, fixing the CI, moving the CI to Github actions and updating out dated documents.

**Comment by michaelbukachi** [comment]
@ulgens Feel free to open any PRs based on open issues than tag me or @caffeinatedMike in the PR .

**Comment by ulgens** [comment]
@michaelbukachi Got it, thank you. Can we start with https://github.com/flask-admin/flask-admin/pull/2069 please? Then I'll send a fix for https://github.com/flask-admin/flask-admin/issues/1978

**Comment by michaelbukachi** [comment]
@ulgens we've already migrated to GitHub actions. See #2169. The tickets you've suggested can be follow up work.

**Comment by ulgens** [comment]
@michaelbukachi Thanks for the info. I didn't realize the config is already in place because the main branch still has `travis.yml` and build badge goes to Travis. Maybe that should be another PR.

**Comment by michaelbukachi** [comment]
@ulgens Yes. We are supposed to do a follow up PR to cleanup.

**Comment by pawl** [comment]
Just saw this thread. It might be safest to remove me as a maintainer for now. I've been doing Django stuff for work for the last few years, and haven't used flask-admin in a long time. (Note: I'm not saying Django's admin is better.)

Maybe we can re-add me in the future if things change and I start working with Flask stuff and contributing to flask-admin again.

**Comment by davidism** [comment]
Flask-Admin is now part of the Pallets Community Ecosystem. Pallets is the open source organization that maintains Flask; Pallets-Eco enables community maintenance of Flask extensions. If you are interested in helping maintain this project, please reach out on the `#pallets-eco` channel on [the Pallets Discord server](https://discord.gg/pallets).

@caffeinatedMike @michaelbukachi @TitaniumHocker @ulgens @odigity
